src/flatbuffers/fb_network_filter.fbs

```diff
@@ -0,0 +1,26 @@
+// flatc --rust --gen-object-api -o src/flatbuffers/ src/flatbuffers/fb_network_filter.fbs
+
+namespace fb;
+
+table NetworkFilter {
+  mask: uint32;  // NetworkFilterMask (network.rs)
+
+  // These arrays contain sorted(asc) indecies in the |unique_domains_hashes|
```

Comment:
indecies  => indices
itself => themselves

Maybe something like:
These arrays contain the sorted (ascending) indices of elements in the |unique_domains_hashes| array, rather than the hashes themselves.